app/assets/javascripts/controllers/SearchCtrl.coffee

```diff
@@ -0,0 +1,29 @@
+@bruseApp.controller 'SearchCtrl', ['FileHandler', '$scope', '$http', (FileHandler, $scope, $http) ->
+  $scope.files = []
+
+  # watches the search input field for changes
+  $scope.$watch "search", () ->
+    if($scope.search != "")
```

Comment:
I like `unless a = b` instead for `if a != b`, but maby that's just me. :hankey: